api-report/sequence.api.md

```diff
@@ -248,6 +257,16 @@ export interface ISerializedInterval {
     start: number;
 }
 
+// @public (undocumented)
+export interface ISerializedIntervalV2 {
```

Comment:
perhaps `ISerializedIntervalCollectionV2`?

Also, if `serializeInternal` is marked properly as `Internal` you should be able to make this internal as well.